package.json

```diff
@@ -41,6 +41,7 @@
     "react-vertical-timeline-component": "3.6.0",
     "react-zoom-pan-pinch": "3.4.3",
     "slick-carousel": "1.8.1",
+    "simplex-noise": "4.0.1",
```

Comment:
Is this used? No updates for 2 years now.

Reply:
Same as above